api/ipcs/client.go

```diff
@@ -39,3 +39,10 @@ func (c *Client) UnpublishBlockchain(blockchainID string) (bool, error) {
 	}, res)
 	return res.Success, err
 }
+
+// GetPublishedBlockchains requests the node to get blockchains being published
+func (c *Client) GetPublishedBlockchains() (*GetPublishedBlockchainsReply, error) {
```

Comment:
I think this should return `([]ids.ID, error)` to be a little more user friendly/inline with other clients.